### ISSUE TITLE
Colin McRae Rally 2.0 - Cubic textures support

### ISSUE THREAD

**Issue by Fabxx** [elishacloud/dxwrapper#88]
The game uses dx7, i've tried the Ddraw compatibility stuff, but it doesn't load further screens on the display, even if the game doesn't crash 

![image](https://user-images.githubusercontent.com/30447649/94907438-776f6680-04a0-11eb-9898-2ff55e279a35.png)

**Comment by elishacloud** [comment]
Yeah, Colin McRae Rally both 1 and 2 require Direct3D.  I have just started adding Direct3D support.  None of the games that require Direct3D will work yet.

**Comment by Fabio-Git-dev** [comment]
ok, thanks

**Comment by elishacloud** [comment]
Ok, I finally have a fix for this.  The latest build works correctly for this game.

Here is the latest build:  [dxwrapper.zip](https://github.com/user-attachments/files/17219520/dxwrapper.zip)

**Comment by Fabxx** [comment]
thank you, will try it as soon as i can and i'll report

**Comment by elishacloud** [comment]
Here is a build with performance profile testing logs.  Can you try playing the level with this build and send me the logs?  This should help me figure out why it is running slow.

[dxwrapper.zip](https://github.com/user-attachments/files/18220094/dxwrapper.zip)

**Comment by elishacloud** [comment]
> i did some tests with silent patch and CUBIC environment mapping settings in the game, i get huge FPS drops in finland stage 2 where there are trees.

dxwrapper's `dd7to9` doesn't support cubic textures yet.  Something I have on my list to add.
